{"name": "创建英雄失败的测试用例-(攻+1)", "status": "passed", "parameters": [{"name": "success_for_name", "value": "'jinx'"}, {"name": "success_for_volume", "value": "1"}, {"name": "fail_for_power", "value": "0"}], "start": 1684150564507, "stop": 1684150564507, "uuid": "1626bccf-c9e8-40d8-93e3-08e21637e4ff", "historyId": "eb63f0828beeba411e782691ddb19ac1", "testCaseId": "e6505b56913fd0a5b5c514cb9e6b3fa7", "fullName": "test_create_hero_homework_01.TestHeroCreate#test_create_hero_fail_for_power_p1", "labels": [{"name": "tag", "value": "run(order=7)"}, {"name": "suite", "value": "test_create_hero_homework_01"}, {"name": "subSuite", "value": "TestHeroCreate"}, {"name": "host", "value": "LAPTOP-0EHG7NSH"}, {"name": "thread", "value": "48660-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_create_hero_homework_01"}]}